crates/nu-command/src/filesystem/ls.rs

```diff
@@ -568,6 +588,180 @@ pub(crate) fn dir_entry_dict(
     Ok(Value::Record { cols, vals, span })
 }
 
+#[cfg(windows)]
```

Comment:
And yeah, what about grouping these windows only functions into a private sub module like 
```rust
#[cfg(windows)]
mod windows_helper {
     pub fn dir_entry_dict_windows_fallback(...) {}
     fn unix_time_from_filetime(...) {}
}
```

Reply:
Great idea, done.